integration_tests.coveragerc

```diff
@@ -0,0 +1,7 @@
+[run]
+
+omit =
+    */tests/*
+    test_*
+
+relative_files = True
```

Comment:
why relative_files is not needed in unit_tests.coveragerc?

Reply:
because relative file view is for merging both unit test coverage and integration.